src/test_transcript.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{Contribution, Transcript};
```

Comment:
You can add `!#[cfg(test)]` to make this whole module only available during test.

Reply:
Yeah will do in next PR – currently it's used by `main`.